tools/scxtop/src/main.rs

```diff
@@ -228,8 +243,9 @@ async fn main() -> Result<()> {
             _ => {}
         }
         0
-    })?;
-    let rb = rbb.build()?;
+    };
+    event_rbb.add(&skel.maps.events, event_handler.clone())?;
```

Comment:
nit: does this need the `.clone()`?

Reply:
Ahh, guess it doesn't right now had left that over from testing mulitple ringbuffers.